client-app/src/examples/contactMobile/details/ContactDetails.scss

```diff
@@ -0,0 +1,48 @@
+.tb-mobile-contact-details-panel {
```

Comment:
This file is not entirely different from the desktop styles, but for mobile I needed CSS animations. Trying to inhert the common parts was proving troublesome due to the nesting.